packages/form/addon/components/cf-field/input/files.hbs

```diff
@@ -24,7 +24,8 @@
           {{file.name}}
         </UkButton>
         <UkIcon
-          class="uk-icon-button uk-margin-small-left"
+          class="uk-icon-button uk-margin-small-left
+            {{if @disabeld 'uk-disabled'}}"
```

Comment:
Could we just hide those buttons altogether if disabled?

Reply:
Sure. Let's do that.